### DISCUSSION TITLE
How long does it take to verify the website after adding the anchor link

### DISCUSSION THREAD

**Discussion by imhamad** [mastodon/mastodon#30194] [Q&A]
Greetings,

I added deknows.com, however keen to know about the time it take to verify

**Comment by StrangeMinds1960** [comment]
What order did you do the two things in?  Did you add the <link rel="me"... to your site first and then do the Mastodon bit, or the other way around.

I found that once I'd verified that the <link rel="me"... was being shown on the site's page source, I created (or recreated) the linked site on Mastodon, this got things working

**Comment by imhamad** [comment]
I added it in similar fashion on my NextJS site

Kindest Regards,
Hamad
________________________________
